bpf-loader/src/eunomia_bpf.cpp

```diff
@@ -512,7 +515,25 @@ bpf_skeleton::get_fd(const char *name) const noexcept
     }
     return -1;
 }
-
+extern "C" {
+const char* libbpf_version_string(void);
+}
+std::string get_eunomia_version() {
```

Comment:
Add tests for the new function in bpf-loader to pass CI

See https://github.com/eunomia-bpf/eunomia-bpf/blob/master/bpf-loader/test/src/config_test.cpp

You may add a new function to test in this file.

Run `make test` in bpf-loader dir to test.